### ISSUE TITLE
Disambiguate selected and non-selected tabs

### ISSUE THREAD

**Issue by BasThomas** [tryswift/trySwiftAppFinal#63]
It would be awesome if we had different tab bar icons for the selected and deselected states.

**Comment by fassko** [comment]
@BasThomas what do you mean with that? Currently selected tab bar icon glows orange. 
![image](https://user-images.githubusercontent.com/29482/40350669-c4d2ab2a-5daa-11e8-97ca-ab2a4c2fc295.png)

**Comment by BasThomas** [comment]
Yes, but for colorblind people that might be hard to distinguish. It would be nice if the icons inverted upon selection.
